internal/xds/translator/listener.go

```diff
@@ -24,6 +24,7 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 
+	gwv1a1 "github.com/envoyproxy/gateway/api/v1alpha1"
```

Comment:
this needs to be removed, we've decoupled the xds layer from the user api layer

Reply:
Fixed.